varats/varats/paper/paper_config.py

```diff
@@ -241,7 +241,7 @@ def load_paper_config(config_path: tp.Optional[Path] = None) -> None:
         if vara_cfg()["paper_config"]["folder"].value is None or \
                 vara_cfg()["paper_config"][
                     "current_config"].value is None:
-            raise Exception(
+            raise RuntimeError(
```

Comment:
```suggestion
            raise LookupError(
```
Maybe lookup error is better for this case, we try to lookup the paper config and cannot find it.

Reply:
According to the [documentation](https://docs.python.org/3/library/exceptions.html#LookupError), `LookupError` is used for missing keys or indices in mappings and sequences. That does not quite match our use case.
The "right" way would be to define our own exception, but i don't think it's worth it.